docs/data-sources/builtin-toolsets/azure-mcp.md

```
@@ -0,0 +1,718 @@
+4. Provide cost breakdown and recommendations
+
+## Troubleshooting
+
+### Authentication Issues
+
+**Problem:** Pod logs show authentication errors
+
+**Solutions:**
+
+1. For Workload Identity: Verify federated identity credentials are configured correctly
+   ```bash
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)